File: eAirFlow.Services/Database/Position.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace eAirFlow.Services.Database;

public partial class Position
{
    public int PositionId { get; set; }

    public string? Name { get; set; }

    public string? Description { get; set; }

    [JsonIgnore]
    public virtual ICollection<Employee> Employees { get; set; } = new List<Employee>();
}

File: UI/eairflow_desktop/lib/models/position.dart
class Position {
  int? positionId;
  String? name;
  String? description;

  Position();

  Position.fromJson(Map<String, dynamic> json) {
    positionId = json['positionId'];
    name = json['name'];
    description = json['description'];
  }

  Map<String, dynamic> toJson() => {
        "positionId": positionId,
        "name": name,
        "description": description,
      };
}
